projects/Gancio/default.nix

```
@@ -0,0 +1,24 @@
+{
+  pkgs,
+  lib,
+  sources,
+}@args:
+{
+  metadata = {
+    summary = "Gancio Shared agenda for local communities that supports Activity";
+      subgrants = [
+        "gancio"
```

Needs to conform to the identifier from the URL from the NLnet project list: https://nlnet.nl/project/Gancio

```suggestion
        "Gancio"
```